integration-tests/tribuo/enclave/build.gradle

```diff
@@ -32,6 +32,8 @@ conclave {
         signingType = dummyKey
     }
 
+    extraJavaModules = "java.logging,java.xml"
```

Comment:
Is it easy to make this config into a list?

Reply:
Probably, I actually followed the same approach we use for `supportLanguages` though, should we be consistent?